kubetest/build.go

```diff
@@ -85,10 +85,29 @@ func (b *buildStrategy) Build() error {
 		target = "quick-release"
 	case "release":
 		target = "release"
+	case "gce-windows-bazel":
+		// bazel doesn't support building multiple platforms simultaneously
+		// yet. We add custom logic here to build both Windows and Linux
+		// release tars. https://github.com/kubernetes/kubernetes/issues/76470
+		// TODO: remove this after bazel supports the feature.
 	default:
 		return fmt.Errorf("Unknown build strategy: %v", b)
 	}
 
+	if target == "gce-windows-bazel" {
```

Comment:
It seems like you mean to either set `target` above, or use `*b` here.  As written I don't understand how this block would get executed

Reply:
Yep, you're right. I wanted to use *b directly and forgot to change. Fixed.